src/test/scala/org/kibanaLoadTest/helpers/ESClient.java

```diff
@@ -77,10 +77,8 @@ public void createIndex(String indexName, Json source) {
             } else {
                 logger.error(String.format("Failed to create index '%s'", indexName));
             }
-        } catch (IOException e) {
-            throw new RuntimeException("IO Exception in esClient.createIndex", e);
-        } catch (ElasticsearchException e) {
-            throw new RuntimeException("createIndex", e);
+        } catch (IOException | ElasticsearchException ex) {
+            throw new RuntimeException(String.format("Failed to create index", indexName), ex);
```

Comment:
The string formatting parameter is missing. This should be `String.format("Failed to create index '%s'", indexName)`